includes/admin/class-wc-stripe-settings-controller.php

```diff
@@ -81,6 +81,11 @@ public function admin_scripts() {
 		];
 		wp_localize_script( 'woocommerce_stripe_admin', 'wc_stripe_settings_params', $params );
 
+		$settings = [
+			'accountStatus' => WC_Stripe_Account::get_account_status(),
```

Comment:
`WC_Stripe_Account` seems to be a singleton in the app - static methods are pretty hard to test.
What are your thoughts about reducing the number of static methods, and having `WC_Stripe_Account` injected in the constructor of `WC_Stripe_Settings_Controller`? That way things will be a bit more testable.